acts-as-taggable-on.gemspec

```diff
@@ -12,7 +12,7 @@ Gem::Specification.new do |gem|
   gem.homepage      = 'https://github.com/mbleigh/acts-as-taggable-on'
   gem.license       = 'MIT'
 
-  gem.files         = `git ls-files`.split($/)
+  gem.files         = `git ls-files`.split($/).select { |f| f.match?(%r{\A((db|lib)/|LICENSE\.md)}) }
   gem.test_files    = gem.files.grep(%r{^spec/})
```

Comment:
IIRC this is not recommended to set anymore and will always be empty in the new implementation

Reply:
Yes, it will be empty after this commit.

Although [the feature still seems to be there](https://github.com/rubygems/rubygems/blob/afbdb6ffb08c9916b260c61a0d4b4f6dcb5bb9af/lib/rubygems/specification.rb), it was [removed from documentation in 2014](https://github.com/rubygems/rubygems/commit/c07356a2de3576bfc5c11192db99a01037524015).